godot-codegen/src/special_cases/special_cases.rs

```diff
@@ -1114,6 +1114,11 @@ pub fn get_derived_virtual_method_presence(class_name: &TyName, godot_method_nam
          | ("PrimitiveMesh", "_get_aabb")
          => VirtualMethodPresence::Override { is_required: false },
 
+         // Methods which are required but not marked as such.
+         // https://docs.godotengine.org/en/stable/classes/class_editorsyntaxhighlighter.html#class-editorsyntaxhighlighter-private-method-create
+         | ("EditorSyntaxHighlighter", "_create")
```

Comment:
I would also mark a related issue in a comment – https://github.com/godot-rust/gdext/issues/1452. There isn't any error message, docs don't mention that it is required etc so it ain't obvious and having more solid reference point might help in the future.